app/src/main/java/pl/pwr/zpi/bcycle/mobile/BCycleNavigationDrawerActivity.kt

```diff
@@ -88,6 +94,13 @@ abstract class BCycleNavigationDrawerActivity: AppCompatActivity() {
             if (it.itemId == R.id.nav_logout) {
                 auth.signOut()
                 finishAndRemoveTask()
+            } else if (it.itemId == R.id.nav_create_group_trip) {
+                startActivity(Intent(applicationContext, TripCreationActivity::class.java))
+                finishAndRemoveTask()
+
+            } else if (it.itemId == R.id.nav_home) {
+                startActivity(Intent(applicationContext, DashboardActivity::class.java))
```

Comment:
The Dashboard should be in the HomeFragment (which is automatically displayed on start and when this option is selected).